src/VideoIn.cs

```diff
@@ -69,7 +73,7 @@ public VideoIn([Pin(Visibility = PinVisibility.Hidden)] NodeContext nodeContext)
 
             try
             {
-                var result = Acquisition.Start(this, device, _logger, _resolution, _fps, _configuration);
+                var result = Acquisition.Start(this, device, _logger, _resolution, _fps, _configuration, _defaultUserSet.ToString() ?? "Default");
```

Comment:
the ToString() cannot return null here -> ?? "Default" can be removed.

Reply:
fixed in 0d98d1e